nibabel/nicom/tests/test_csareader.py

```diff
@@ -14,6 +15,8 @@
 
 from .test_dicomwrappers import (have_dicom, dicom_test,
                                  IO_DATA_PATH, DATA, DATA_FILE)
+if have_dicom:
+    from .test_dicomwrappers import pydicom
```

Comment:
I ended up keeping the top level import and switching the other function to use it. This way the import logic doesn't get replicated and I can use the pydicom version in the `skipif` decorator.